hw/drivers/sensors/mpu6050/src/mpu6050_priv.h

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef __MPU6050_PRIV_H__
+#define __MPU6050_PRIV_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum mpu6050_registers {
+    MPU6050_SELF_TEST_X = 0x0d, // RW
```

Comment:
Our coding style does not allow c++ style comments, hence all the "// RW and //R " comments should either be converted to C comments which is "/* RW */ and /* R */ or removed.